packages/prettier-config/index.js

```diff
@@ -1,8 +1,3 @@
-module.exports = {
-  singleQuote: true,
-  arrowParens: "avoid",
-  semi: false,
-  proseWrap: "always",
-  printWidth: 100,
-  trailingComma: "es5",
-};
+import config from './.prettierrc.json' assert { type: 'json' }
```

Comment:
this is an experimental option but otherwise we'd need to duplicate the prettier config